docs/data/date-pickers/legacy-date-time-picker/legacy-date-time-picker.md

```diff
@@ -0,0 +1,67 @@
+---
+product: date-pickers
+title: React Date Time Picker component
+components: DateTimePicker,DesktopDateTimePicker,MobileDateTimePicker,StaticDateTimePicker
+githubLabel: 'component: DateTimePicker'
+packageName: '@mui/x-date-pickers'
+materialDesign: https://m2.material.io/components/date-pickers
+---
+
+# Date Time Picker
+
+<p class="description">This component combines the date & time pickers.</p>
+
+:::warning
+This page describes how the legacy date time pickers work.
+If you are already using the v6 alpha or beta, we recommend you to try the [new date time pickers](/x/react-date-pickers/date-time-picker/).
+
+At the end of the v6 beta phase, the legacy date time pickers will be removed and replaced the new date time pickers
+(which will be renamed to match the current name of the legacy pickers).
+:::
+
+It allows the user to select both date and time with the same control.
+
+Note that this component is the [DatePicker](/x/react-date-pickers/legacy-date-picker/) and [TimePicker](/x/react-date-pickers/legacy-time-picker/)
```

Comment:
```suggestion
Note that this component is the [NextDatePicker](/x/react-date-pickers/date-picker/) and [NextTimePicker](/x/react-date-pickers/time-picker/)
```

Reply:
We are on the legacy page so it's not the combination of the `Next` pickers